package.json

```diff
@@ -1,10 +1,18 @@
 {
   "name": "@politico/vue-accessible-selects",
-  "version": "0.0.1",
+  "version": "0.0.2-alpha.5",
   "description": "Select & Multi Select implementations for Vue, focused especially on implementing accessibility best practices",
-  "main": "index.js",
+  "files": [
+    "dist"
+  ],
+  "main": "dist/index.js",
```

Comment:
If your only output is es modules, I suggest using `module` here. 

https://rollupjs.org/guide/en/#publishing-es-modules

Reply:
Great call, will switch...